src/index.ts

```diff
@@ -87,6 +89,8 @@ export * from './flow-spec/SupportedMode'
 export * from './model/block/IBlockConfig'
 export * from './model/block/ICaseBlock'
 export * from './model/block/ICaseBlockConfig'
+export * from './model/block/IDirectorySelectionBlock'
+export * from './model/block/IDirectorySelectionBlockConfig'
 export * from './model/block/ILogBlock'
 export * from './model/block/ILogBlockConfig'
 export * from './model/block/IMessageBlock'
```

Comment:
should the same be added to the `dist/index.d.ts` or is it auto-generated?

(right now writing `import {DirectorySelectionPrompt} from '@floip/flow-runner'` gives an error)

Reply:
@AlteaDown @bzabos